xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/resources/ApplicationResources.properties

```diff
@@ -91,8 +91,9 @@ livedata.pagination.last=Last page
 
 livedata.action.refresh=Refresh
 livedata.action.addEntry=Add entry
-livedata.action.reorder.hint=Drag to reorder
-livedata.action.resizeColumn.hint=Drag to resize. Double click to reset.
```

Comment:
Sorry for noticing this only now, but this is not how we handle translation updates, see https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/#HUpdatingtranslations - the old keys need to be deprecated instead of being removed, and new keys need to be added for the updated messages.

Reply:
I updated translation to follow our translation update practices. 
I tried to be as thorough as possible, but I can't provide correct translations for most languages (just gave some for french). Instead I just took the former value, which had part of what should be in the value, and went with that. This is not perfect but I guess it's still better than putting english.

I looked at https://github.com/xwiki/xwiki-platform/blob/539f97b0d6f8f9e049abe2a0c5321e41a137ffa0/xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties#L21C1-L43C80 and tried to follow the pattern described and used there.

Changes were made in https://github.com/xwiki/xwiki-platform/pull/2475/commits/e9a58de555032dd6282d1361895eb99b644b6d3a (changing the translations) and https://github.com/xwiki/xwiki-platform/pull/2475/commits/932801e2e7440450c6c4c1c02f42f64ade0df843 (adding the commentary section to explain organization of the translations).

Reply:
You should normally not update other languages. I think the sentence 

> When deprecating a key in favor of a different one, you should try to see if some of the existing translations can be copied over as well.

Just says that if you, e.g., renamed an XClass property and now need to rename the translation keys as well, you should also see if you can copy the translations such that they don't need to be translated again. However, I think you shouldn't copy "wrong" translations as then translators won't see the strings as missing and cannot update them.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2475/commits/8fb8cb945fa970eda65207236cf5885eab05d5fd :+1: